x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/risk_score_mapping/default_risk_score.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiFormRow, EuiText, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiRange } from '@elastic/eui';
+import type { EuiRangeProps } from '@elastic/eui';
+import { MIN_RISK_SCORE, MAX_RISK_SCORE } from './constants';
+import * as i18n from './translations';
+
+interface DefaultRiskScoreProps {
+  value: number;
+  onChange: (newValue: number) => void;
+  errorMessage?: string;
+  idAria?: string;
+  dataTestSubj?: string;
+}
+
+export function DefaultRiskScore({
+  value,
+  onChange,
+  errorMessage,
+  idAria,
+  dataTestSubj = 'defaultRiskScore',
```

Comment:
Upon investigation I decided to skip this one since I'll then have to make the prop values more generic and update a bunch of places on the Rule Editing page and in tests for consistency. That's going to be a lot of changes.